eclipse/plugin.xml

```diff
@@ -956,71 +956,27 @@
       <menuContribution
             allPopups="false"
             locationURI="popup:org.eclipse.ui.popup.any?after=additions">
-            <menu
-                  icon="icons/elcl16/session_tsk.png"
-                  label="Share With"
-                  tooltip="Share the Selected Projects With Your Contacts">
-                  <visibleWhen
+         <command
+               commandId="saros.ui.commands.ShareResource"
+               disabledIcon="icons/dlcl16/spacer.png"
```

Comment:
Is there a reason to use this spacer instead of the real icon when the menu entry is disabled?

Reply:
I simply used the same configuration as in the main menu. I am not 100% when the disabledIcon is used. If our constrains (as a running session has to exist) are not satisfied, [the normal icon will simply appear greyed out](https://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fextension-points%2Forg_eclipse_ui_viewActions.html):

![screenshot](https://user-images.githubusercontent.com/7346610/57840137-57b2a200-77c8-11e9-93af-941a11090d92.png)